### DISCUSSION TITLE
DWIN T5UID1 / Ender 5 Plus with Fysetc S6 (STM32F446)

### DISCUSSION THREAD

**Discussion by Anima1000000** [Desuuuu/DGUS-reloaded-Klipper#32] [Q&A]
I am trying to get the TSUID1 display working in combination with a Fysetc S6 v2.1 (STM32F446). I had it working before on the orginal Creality 4.2.2 board but on the S6 I just see the Klipper logo.

In the make menuconfig I have selected USART3 which I think should correspond with pins PC10 USART-TX3 and PC11 USART-RX3.
I have hooked the the display TX to the RX pin on the S6 board and vice versa.

I have tried both USART3 options. In klippy.log I noticed this line `RESERVE_PINS_t5uid1=PB11,PB10` Should that be set to PC11,PC10?

Is there a way to check what pins have been assigned while flashing the board?

![image](https://user-images.githubusercontent.com/42918730/168445443-0ca14077-3c41-4d8b-9de4-b90ec0c45a85.png) 

Thanks for the work you have put in, in making the TSUID1 display in Klipper.

**Comment by Anima1000000** [comment]
I got it to work over USART1 with the [dgus-display branch](https://github.com/Desuuuu/klipper/tree/dgus-display).

However also there the USART3 pins do not match those on the schematic.
This is from klippy.log:
```
BUS_PINS_usart2=[_],PA3,PA2 BUS_PINS_usart3=[_],PB11,PB10 BUS_PINS_usart1=[_],PA10,PA9
BUS_PINS_usart1a=[usart1],PB7,PB6 BUS_PINS_usart3a=[usart3],PD9,PD8 BUS_PINS_usart2a=[usart2],PD6,PD5
```
I would expect USART3 to be PC10,PC11

**Reply by Desuuuu** [reply]
Both are valid, as this chip has fairly good multiplexing capabilities.

This might not be valid for your particular board layout though.
